TF-15131 Update product documentation with new default project behavior (#689)
hashicorp/terraform-docs-common: website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx

#### General

> Review the organization name and contact email. Organization owners can choose to change the organization name, contact email, default execution mode, and the default project, or delete the organization. When an organization owner updates the default execution mode, all workspaces configured to [inherit this value](/terraform/cloud-docs/workspaces/settings#execution-mode) will be affected.

Reviewer: ```suggestion
Review the organization name and contact email. Organization owners can choose to change the organization name, contact email, default execution mode, and the default project, or delete the organization. When an organization owner updates the default execution mode, all workspaces configured to [inherit this value](/terraform/cloud-docs/workspaces/settings#execution-mode) are affected.
```

#### General

> Review the organization name and contact email. Organization owners can choose to change the organization name, contact email, default execution mode, and the default project, or delete the organization. When an organization owner updates the default execution mode, all workspaces configured to [inherit this value](/terraform/cloud-docs/workspaces/settings#execution-mode) are affected.

Reviewer: ```suggestion
Review the organization name and contact email. Organization owners can change the organization's name, contact email, default execution mode, designated default project, or can delete the organization. When an organization owner updates the default execution mode, all workspaces that [inherit that value](/terraform/cloud-docs/workspaces/settings#execution-mode) will also inherit that execution mode.
```
